libs/net/dal/Migrations/1.2.4/Up/PostUp/00-ReportTemplate.sql

```diff
@@ -0,0 +1,355 @@
+DO $$
+BEGIN
+
+-- Update custom report with latest template.
```

Comment:
Fixed duplicate image file name issue.  Refer to the cshtml file to see the change.